### PR TITLE
RichCommand: Refresh items

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- Implements rich command for refresh

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [x] Did you remove any strings from the en-us resource file?
   - [x] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open a folder and click the refresh button in the address bar
   2. Use the `ctrl + r` keyboard shortcut
   3. Right click in a blank space and verify the refresh option works

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 1]
```diff
@@ -113,15 +113,12 @@
...
+					Command="{x:Bind Commands.RefreshItems, Mode=OneWay}"
+					IsEnabled="{x:Bind Commands.RefreshItems.IsExecutable, Mode=OneWay}"
 					Style="{StaticResource AddressToolbarButtonStyle}"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=NavRefreshButton/ToolTipService/ToolTip}">
-					<FontIcon FontSize="14" Glyph="&#xE72C;" />
-					<Button.KeyboardAccelerators>
-						<KeyboardAccelerator Key="F5" />
-					</Button.KeyboardAccelerators>
+					ToolTipService.ToolTip="{x:Bind Commands.RefreshItems.Label}">
+					<FontIcon FontSize="14" Glyph="{x:Bind Commands.RefreshItems.Glyph}" />
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:121]: The icon isn't displaying properly

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 2]
```diff
@@ -113,15 +113,12 @@
 					x:Name="Refresh"
 					x:FieldModifier="public"
 					AccessKey="R"
-					AutomationProperties.Name="{helpers:ResourceString Name=NavResfreshButton/AutomationProperties/Name}"
-					Command="{x:Bind ViewModel.RefreshClickCommand, Mode=OneWay}"
-					IsEnabled="{x:Bind ViewModel.CanRefresh, Mode=OneWay}"
+					AutomationProperties.Name="{x:Bind Commands.RefreshItems.Label}"
+					Command="{x:Bind Commands.RefreshItems, Mode=OneWay}"
+					IsEnabled="{x:Bind Commands.RefreshItems.IsExecutable, Mode=OneWay}"
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:118]: The button is disabled when it shouldn't be

File: src/Files.App/Actions/Content/RefreshItemsAction.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	internal class RefreshItemsAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label { get; } = "Refresh".GetLocalizedResource();
+
+		public RichGlyph Glyph { get; } = new("\uE72C");
+
+		public HotKey HotKey { get; } = new(VirtualKey.R, VirtualKeyModifiers.Control);
+
```
Comment (ferrariofilippo) [inline src/Files.App/Actions/Content/RefreshItemsAction.cs:22]:
```suggestion

        public HotKey SecondHotKey { get; } = new(VirtualKey.F5);
        
```

**Comment by ferrariofilippo** [comment]
Yair in `ContextFlyoutItemHelper.cs` you can use `ContextMenuFlyoutItemViewModelBuilder()` as following:

File: src/Files.App/Actions/Content/RefreshItemsAction.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+		}
+
+		public async Task ExecuteAsync()
+		{
+			context.ShellPage?.Refresh_Click();
+		}
+
+		private void Context_PropertyChanged(object? sender, PropertyChangedEventArgs e)
+		{
+			if (e.PropertyName is nameof(IContentPageContext.ShellPage.ToolbarViewModel.CanRefresh))
```
[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Actions/Content/RefreshItemsAction.cs:38]: That's a good point, we should keep this then.

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Actions/Content/RefreshItemsAction.cs]: I implemented this change, can you take a look?

File: src/Files.App/Strings/en-US/Resources.resw [hunk 5]
```diff
@@ -243,9 +243,6 @@
   <data name="Ascending" xml:space="preserve">
     <value>Ascending</value>
   </data>
-  <data name="BaseLayoutContextFlyoutRefresh.Text" xml:space="preserve">
```
[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Strings/en-US/Resources.resw:246]: The text was changed as well so it'll need to be retranslated either way.

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Strings/en-US/Resources.resw:246]: We used to show the tooltip as well

**Pull request merged**